Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -196,11 +209,11 @@ def create_quam_superconducting(
 if __name__ == "__main__":
     folder = Path(__file__).parent
     quam_folder = folder / "quam_state"
-
+    # TODO: what is this for? WHy not specifying OPX+/Octave/LF/MW in the wiring?
     quam_class = FEMQuAM
```

Comment:
It is so that our IDE knows what type of QuAM we're going to be using. Otherwise the IDE has no idea what `create_superconducting_quam` is going to return